src/constants/errors.ts

```diff
@@ -0,0 +1,11 @@
+export const UNKNOWN_INTERACTION = {
```

Comment:
you can make an Object ERROR and within that, you can make a key of the error type. This would be  easier to read and maintain

Reply:
I am importing it as `import * as error from errors.ts` then I am using it as `error.UNKNOWN_INTERACTION`

Reply:
okay. You are making constant files separately. Ideally, it should be one file only. Any reason for making it constants/errors?

Reply:
it will segregate different constants as in the organization we are moving towards a centralized repository that will contain all constants thus the separate constants files for URLs, interactions and errors made a better sense to me.

Reply:
okay. I am not aware of any org-level decision. If this is so, then it is fine.